requirements/dev.txt

```diff
@@ -10,8 +10,6 @@ amqp==5.0.6
     # via
     #   -r requirements/requirements.txt
     #   kombu
-appnope==0.1.2
-    # via ipython
```

Comment:
This went away after running `hdev requirements`

Based on this I assume it would appear/disappear depending on which platform we run it from

https://github.com/ipython/ipython/blob/master/setup.py#L213